src/libs/nav.ts

```diff
@@ -73,13 +73,19 @@ export const getPath = (name: string, params?: Record<string, any>): string => {
 /**
  * alias for getPath()
  */
-export const getLink = (name: string, params?: Record<string, any>) => `#${getPath(name, params).slice(1)}`; // slice off leading slash
+export const getLink = (name: string, params?: Record<string, any>) =>
```

Comment:
For a clearer interface, let's separate the query parameters from the URL parameters.

```suggestion
export const getLink = (name: string, urlParams?: Record<string, any>, queryParams?: Record<string, any>) =>
```